### PR TITLE
Test usability fix

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Having the dummy data in the past causes debugging pain

### CODE REVIEW

**Comment by seamuslee001** [comment]
@eileenmcnaughton 

```
not ok 3019 - Failure: CRM_Utils_TokenConsistencyTest::testEventTokenConsistency
  ---
  message: 'event.end_date :October 23rd, 2008 .  not found in  MIME-Version: 1.0'
  severity: fail
  ...
  ```

**Comment by eileenmcnaughton** [comment]
@seamuslee001 it passed

File: tests/phpunit/CiviTest/CiviUnitTestCase.php [hunk 1]
```diff
@@ -1119,10 +1119,10 @@ public function eventCreate(array $params = []): array {
       'event_type_id' => 1,
       'is_public' => 1,
       'start_date' => 20081021,
-      'end_date' => 20081023,
+      'end_date' => '+ 1 month',
```
Comment (seamuslee001) [inline tests/phpunit/CiviTest/CiviUnitTestCase.php:1122]: so a 6 year event? :)

Comment (eileenmcnaughton) [reply tests/phpunit/CiviTest/CiviUnitTestCase.php:1122]: @seamuslee001 yeah - key thing is the end date is in the future & start date is in the past so when you try to use it you don't get told it is invalid

**Comment by colemanw** [comment]
Better than it was before anyway.

**Pull request merged**